CHANGELOG.md

```diff
@@ -1,29 +1,37 @@
 # bedrock-mongodb ChangeLog
 
-## [Unreleased]
+## Unreleased
 
-## [2.1.4] - 2016-05-23
+### Changed
+- Add `config.mongodb.local.enable` option to enable using a local database.
+  Disabled by default. In a remote replicated database setup, the local
+  database needs to actually be local. Running a full mongodb node is
+  heavyweight for the purposes of this module. For the time being, if the local
+  database is disabled, every startup of a server will get a new globalId.  A
```

Comment:
Every new process will get a new globalId. (not every server -- the definition of that is ambiguous)